src/aiida/cmdline/commands/cmd_profile.py

```diff
@@ -165,3 +165,45 @@ def profile_delete(force, delete_data, profiles):
 
         get_config().delete_profile(profile.name, delete_storage=delete_data)
         echo.echo_success(f'Profile `{profile.name}` was deleted.')
+
+
+@verdi_profile.command('dbdump')
```

Comment:
Why is this command here? This is profile specific, not all profiles will have a PSQL database.

Reply:
thanks for noticing, was part of initial testing, not sure why it still remained, i think i already removed it. will remove again.